pmonitor/pmon

```diff
@@ -21,7 +21,7 @@ while [ "$1" != "" ]; do
   shift;
 done;
 
-export PID_DIR=${WORKDIR}/pid_monitor
+export PID_DIR=${WORKDIR}/perftools-setup/pid_monitor
```

Comment:
This is the place below, so seems there should not be a need to modify the code, I have destroyed my cluster, will revisit this when we plan to re run the code.

# Steps to install pid_monitor.
if [ ! -d ${WORKDIR}/pid_monitor ]; then
	git clone https://github.com/jschaub30/pid_monitor

	mkdir -p ${WORKDIR}/pid_monitor/rundir

	#Set execute permission for others so that apache can read files from this directory.
	sudo chmod o+x ${WORKDIR} ${WORKDIR}/pid_monitor ${WORKDIR}/pid_monitor/rundir
fi